src/server.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Server start point
+ */
+require('require-self-ref')
```

Comment:
I hope everyone is cool with this being added. It makes `require` so much better.